<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ForgetDialog</class>
 <widget class="QDialog" name="ForgetDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>562</width>
    <height>378</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>55</y>
     <width>54</width>
     <height>12</height>
    </rect>
   </property>
   <property name="text">
    <string>用户名 </string>
   </property>
  </widget>
  <widget class="QLineEdit" name="userLineEdit">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>50</y>
     <width>341</width>
     <height>20</height>
    </rect>
   </property>
   <property name="placeholderText">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>120</y>
     <width>48</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>密保问题</string>
   </property>
  </widget>
  <widget class="QComboBox" name="questionComboBox">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>103</y>
     <width>341</width>
     <height>20</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>您的所属单位？</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>您的生日？</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>您的姓名？</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>您所上的大学？</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>您父母的生日？</string>
    </property>
   </item>
  </widget>
  <widget class="QLineEdit" name="questionLineEdit">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>129</y>
     <width>341</width>
     <height>20</height>
    </rect>
   </property>
   <property name="placeholderText">
    <string>正确回答密保问题才能变更新密码</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>238</y>
     <width>60</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>重复新密码</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>182</y>
     <width>36</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>新密码</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="passwdLineEdit">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>180</y>
     <width>341</width>
     <height>20</height>
    </rect>
   </property>
   <property name="echoMode">
    <enum>QLineEdit::Password</enum>
   </property>
   <property name="placeholderText">
    <string>请输入新密码，长度为8-16位</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="passwdAgainLineEdit">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>237</y>
     <width>341</width>
     <height>20</height>
    </rect>
   </property>
   <property name="echoMode">
    <enum>QLineEdit::Password</enum>
   </property>
   <property name="placeholderText">
    <string>请重复输入新密码</string>
   </property>
  </widget>
  <widget class="QPushButton" name="cancelButton">
   <property name="geometry">
    <rect>
     <x>345</x>
     <y>299</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 8pt &quot;微软雅黑&quot;;</string>
   </property>
   <property name="text">
    <string>取消</string>
   </property>
  </widget>
  <widget class="QPushButton" name="okButton">
   <property name="geometry">
    <rect>
     <x>121</x>
     <y>299</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 8pt &quot;微软雅黑&quot;;</string>
   </property>
   <property name="text">
    <string>确定</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>cancelButton</sender>
   <signal>clicked()</signal>
   <receiver>ForgetDialog</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>370</x>
     <y>334</y>
    </hint>
    <hint type="destinationlabel">
     <x>475</x>
     <y>335</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
